doc/users/plotting/colormaps/lightness.py

```diff
@@ -1,120 +1,100 @@
 '''
+==========================
+Lightness of the colormaps
+==========================
+
 For each colormap, plot the lightness parameter L* from CIELAB colorspace
 along the y axis vs index through the colormap. Colormaps are examined in
 categories as in the original matplotlib gallery of colormaps.
 '''
 
-from colormaps import cmaps
 import numpy as np
+import matplotlib as mpl
 import matplotlib.pyplot as plt
 from matplotlib import cm
-import matplotlib as mpl
 from colorspacious import cspace_converter
+from colormaps import cmaps  # the colormaps, grouped by category
 
 mpl.rcParams.update({'font.size': 12})
 
-# indices to step through colormap
+# Number of colormap per subplot for particular cmap categories
+_DSUBS = {'Perceptually Uniform Sequential': 4, 'Sequential': 6,
+          'Sequential (2)': 6, 'Diverging': 6, 'Qualitative': 4,
+          'Miscellaneous': 6}
+
+# Spacing between the colormaps of a subplot
```

Comment:
I do not understand to which situation you are refering to: looking at [blame](https://github.com/afvincent/matplotlib/blame/a769e5b3afe317f8626738f611c33bfb29f2b6b8/doc/users/plotting/colormaps/lightness.py#L23) on GH, the `dc` and `dsub` strategy is used since at least 3 years. I tried to not modify the spirit of the plots, but rather to refactor what could be, and to make the magic values a bit more uniform across all the different cases.